### PR TITLE
Add Interhost Networks Company to the list

### PR DESCRIPTION
**Pull request by interhost7**

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "interhost.co.il",
+    "link": "https://www.interhost.co.il",
+    "category": "full",
```
Comment (bmw) [inline _data/hosting_providers.json:5]: I think this should be partial based on our descriptions at https://certbot.eff.org/hosting_providers/ and the tutorial you included.
```suggestion
    "category": "partial",
```

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    "tutorial": "https://interhost.co.il/ssl-installation/",
+    "announcement": "",
+    "plan": "https://interhost.co.il/servers-services/cloud-services/virtual-servers/",
```
Comment (bmw) [inline _data/hosting_providers.json:8]: Per https://github.com/certbot/website#hosting-providers-list, this link isn't going to show up so I think we should just remove it.
```suggestion
    "plan": "",
```

**Review by bmw — CHANGES_REQUESTED** [review]

**Comment by interhost7** [comment]
Hi, 
Thank you for the suggestions. We have fixed the fields, please review again.

**Review by bmw — APPROVED** [review]
Great. Thanks! This LGTM.

**Pull request merged**